trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    super[0, 10]
```

Comment:
- [ ] Please read my above comment carefully and also implement the behavior of this class as instructed in the `milestone` guidelines.